src/avram/define_attribute.cr

```diff
@@ -100,30 +100,7 @@ module Avram::DefineAttribute
 
     {% name = key.id %}
 
-    @_{{ name }} : Avram::Attribute(Avram::Uploadable?)?
-
-    ensure_base_attributes_method_is_present
-
-    def attributes
-      ([{{ name }}] + previous_def + super).uniq
-    end
-
-    def {{ name }}
-      _{{ name }}.permitted
-    end
-
-    private def _{{ name }}
-      @_{{ name }} ||= Avram::Attribute(Avram::Uploadable?).new(
-        name: :{{ name }},
-        param: {{ name }}_param,
-        value: nil,
-        param_key: self.class.param_key
-      ).tap do |attribute|
-        if {{ name }}_param_given?
-          set_{{ name }}_from_param(attribute)
-        end
-      end
-    end
+    attribute {{ name }} : Avram::Uploadable
```

Comment:
These methods were all duplicated from `attribute` anyway. The only thing this did different was change how `{{ name }}_param` and `{{ name }}_param_given?` worked.